ramls/fund_distribution.raml

```diff
@@ -22,7 +22,7 @@ resourceTypes:
     collection-item: !include raml-util/rtypes/item-collection.raml
 
 
-/fund_distribution:
+/orders-storage/fund_distribution:
```

Comment:
Along with adding `orders-storage` prefix the approach suggested in [MODORDSTOR-18](https://issues.folio.org/browse/MODORDSTOR-18) is to use plural form for the collection resources. So it makes sense to use `/orders-storage/fund_distributions`

Reply:
Changed to /orders-storage/fund_distribution**s**  according to REST naming conventions